docs/osf-trivia-list/dasharo.md

```diff
@@ -154,3 +154,83 @@ Dasharo is not coreboot firmware distribution because we also may imagine
+    - [clean and simple code](#dasharo-clean-and-simple-code),
+    - [long-term maintenance](#dasharo-long-term-maintenance),
+    - [professional support](#dasharo-support-package),
+    - [transparent validation](#dasharo-transparent-validation),
+    - [privacy-respecting implementation](#dashro-privacy-respecting-implementation),
+    - [liberty for the owners](#dasharo-liberty-for-the-owners),
+    - [trustworthiness for all](#dasharo-trustworthiness-for-all).
+
+### Dasharo Zero-Touch Initial Deployment
+
+Documentation supported hardware provide information about initial deployment,
+updates and recovery procedures. Developed by Dasharo Team [Dasharo Tools Suite
+(DTS)][dtsos] operating system support users by automating the deployment
+process, which help reducing errors and inconsistencies, and make sure the
+firmware can be further updated to new version without any problems. DTS also
+provide controlled and secure environment for initial deployment and update of
+firmware, reducing the risk of tampering or unauthorized changes.
+
+Dasharo Zero-Touch Initial Deployment is smooth, effortless and user-friendly
+process, which reduce user frustration and improve satisfaction.
+
+Use of DTS largely improve firmware adoption, hardware compatibility reporting
+and binary blobs transmission, as well as recovery.
+
+For more details about zero-touch initial deployment please read relevant
+Dasharo Tools Suite
+[documentation](../../dasharo-tools-suite/documentation/#dasharo-zero-touch-initial-deployment).
+
+### Dasharo Clean and Simple Code
+
+* Each supported platform has its own branches in Dasharo's coreboot repository:
+    - `<board_name>/develop`, e.g. `msi_ms7d25/develop`, used for bug fixes and
+    development of new features.
+    - `<board_name>/release`, e.g. `msi_ms7d25/release`, contains only the
+    released and validated code. It is used to build the firmware binaries
+    distributed to the community through Dasharo releases.
+* Each supported platform has a build scripts included in Dasharo's coreboot
+  repository to simplify the complex build instructions
+
+### Dasharo Long Term Maintenance
+
+* We provide long term maintenance - coreboot community for various reasons, do
+  not merge some patches, because of understaffing, lack of reviewers. Some
+  changes have long way to upstream, we maintain those patches and make them
+  work before those will go upstrea. If ever, we are committed to maintain
+  platforms which are moved to branch in coreboot.
+* Firmware update - we are registered [consultants for fwupd/LVFS][lvfs] and
+  enable customers and community platforms, so they can get seamless firmware
+  update in Linux.
+
+### Dasharo Support Package
+
+* Support - Dasharo Team is paid to provide support in the community through
+  [Dasharo Matrix Space][dasharo-matrix], of course we want to introduce some
+  commercial products related to the support to make the business feasible, but
```

Comment:
We have such product, and it is named exactly as the name of this section: `Dasharo Support Package`. What is described here, is not DSP.